src/index 2.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
why is there a second index file here?

Reply:
I did not notice it. Im not sure why it is in the directory. It does not seem to be imported in any other file.
Let me know if I should delete it or keep it

Reply:
what purpose is it serving? It doesn't seem like its doing anything so please delete it